package protoFile.campaign.ObtainJiangLi4401;
message ObtainJiangLiRequest{
     required int32 id=1;//½ÇÉ«id
}
message ObtainJiangLiResponse{
     optional string message=1;
     optional bool result=2;
}